app/scripts/controllers/network/network-controller.test.js

```diff
@@ -1,7 +1,9 @@
 import sinon from 'sinon';
-import { getNetworkDisplayName } from './util';
+import { NETWORK_TO_NAME_MAP } from '../../../../shared/constants/network';
 import NetworkController, { NETWORK_EVENTS } from './network';
 
+const getNetworkDisplayName = (key) => NETWORK_TO_NAME_MAP[key];
```

Comment:
This function was the last remaining thing in the network utils module, and it was only used in this test. I've moved it here instead so we can delete that module.